torchvision/utils.py

```diff
@@ -191,34 +193,45 @@ def draw_bounding_boxes(
     elif image.size(0) not in {1, 3}:
         raise ValueError("Only grayscale and RGB images are supported")
 
+    if labels:
+        if len(labels) != boxes.size(0):
+            raise ValueError("Number of boxes and labels mismatch. Please specify labels for each box.")
+
+    # List of colors should have minimum colors.
+    if colors:
+        if isinstance(colors, list):
+            if len(colors) < boxes.size(0):
+                raise ValueError("Number of colors should be greater or equal to the number of boxes.")
```

Comment:
This is used to silently throw list index out of range error, if number of colors were less than boxes.
I guess it's better to throw an error.